packages/cli/vitest.config.ts

```diff
@@ -4,9 +4,9 @@ import { readFileSync } from 'node:fs';
 export default defineConfig({
   plugins: [
     {
-      name: 'markdown-loader',
+      name: 'file-loader',
       transform(_src, id) {
-        if (id.endsWith('.md')) {
+        if (id.endsWith('.md') || id.endsWith('.sh') || id.endsWith('.yml')) {
```

Comment:
I think this is fine just because we are not currently testing those cases directly. The current configuration loads `yml` files as assets, which means that the import returns the path and not the content.

https://github.com/endorhq/rover/blob/6d118316450884b68db83da90b84d1b364d99f9b/packages/cli/tsdown.config.ts#L25

Based on that config, the following import will return the file `path` and not the content.

```js
import filePath from "./file.yml";
```

However, we can fix this in a separate PR. Just something to keep in mind.

Reply:
Updated :)